client/src/components/AuthMenu/AuthMenu.tsx

```diff
@@ -25,9 +33,18 @@ const AuthMenu = (): JSX.Element => {
 
   return (
     <div>
+      {/*
       <IconButton aria-label="show auth menu" aria-controls="auth-menu" aria-haspopup="true" onClick={handleClick}>
         <MoreHorizIcon />
-      </IconButton>
+      </IconButton>*/}
+      <Avatar
+        aria-label="show auth menu"
+        aria-controls="auth-menu"
+        aria-haspopup="true"
+        onClick={handleClick}
+        alt="Profile Image"
+        src={`https://robohash.org/${user.email}.png`}
+      />
```

Comment:
This looks good 👍🏼 Alternatively, we could have modified/passed props to the `AvatarDisplay` component in the starter code to implement this.